test/new-e2e/agent-subcommands/status_test.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package sharedcomponents
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/e2e"
+	"github.com/stretchr/testify/assert"
+)
+
+type commandStatusSuite struct {
+	e2e.Suite[e2e.AgentEnv]
+}
+
+func TestStatusSuite(t *testing.T) {
+	e2e.Run(t, &commandStatusSuite{}, e2e.AgentStackDef(nil))
+}
+
+func (v *commandStatusSuite) TestStatusNotEmpty() {
+	err := v.Env().Agent.WaitForReady()
+	assert.NoError(v.T(), err)
+
+	status := v.Env().Agent.Status()
+	assert.NotEmpty(v.T(), status.Content)
+}
```

Comment:
suggestion: this test seems a little redundant with the following `TestNoRenderError`, perhaps we could call `datadog-agent stop` and assert there is a 255 exit code returned and restart the service?

Reply:
Yes you're right ! 

Unfortunately I don't think we have easy way to restart the Agent. From the code in test-infra-definitions, we store the [systemctl restart command](https://github.com/DataDog/test-infra-definitions/blob/dabbb1adc2dfea23859418212861b028d8f6d6d3/components/os/servicemanager.go#L9-L13) which should be usable with something like `vm.GetOs().GetServiceManager().RestartAgentCmd()` and then executed with pulumi (see [code](https://github.com/DataDog/test-infra-definitions/blob/dabbb1adc2dfea23859418212861b028d8f6d6d3/components/datadog/agent/install.go#L69-L80)). I believe it a bit too complex atm and I'd rather ask the squad about the best way to do it imho

I'll remove it and follow up later to add the test when the Agent is not running since I want to unblock your PRs, WDYT ?